main.py

```diff
@@ -0,0 +1,69 @@
+"""Proxy server to communicate a client to ARTIQ."""
+
+from contextlib import asynccontextmanager
+import json
+import os
+
+from fastapi import FastAPI
+from sipyco.pc_rpc import Client
```

Comment:
I think `Client` is too general name.. How about importing the module instead?

For example:
```python
from sipyco import pc_rpc
```
or
```python
from sipyco import pc_rpc as rpc
```
if you don't like the underscore in a module name.

Reply:
Good! I will apply the latter one.